specifyweb/backend/trees/views.py

```
@@ -241,10 +242,22 @@ def get_tree_rows(treedef, tree, parentid, sortfield, include_author, include_st
         .outerjoin(synonym, synonym.AcceptedID == node._id)
         .where(treedef_col == int(treedef))
         .where(node.ParentID == parentid)
-        .group_by(node._id)
-        .order_by(orderby)
     )
 
+    if biostrat != 'all' and tree == 'geologictimeperiod':
+        if biostrat == 'bio':
+            query = query.where(
+                (node.ParentID == None) | (node.isBioStrat == True)
```

Ruff (0.15.20)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)